vortex-sampling-compressor/src/lib.rs

```diff
@@ -173,6 +200,13 @@ impl<'a> SamplingCompressor<'a> {
         cloned
     }
 
+    pub fn only(&self, compressors: &[CompressorRef<'a>]) -> Self {
```

Comment:
except that it doesn't clear disabled_compressors, so we can disable stuff further up the tree (e.g., when already used) but be explicit about what the allowed set is further down the tree

Reply:
This method is never called. I think we should remove it and probably also change semantics of including. It's weird that you can include something and it not be included because it's been disabled earlier. At the same time I think if we see usages of this method there's likely a better way of doing things.